molecule/default/verify.yml

```diff
@@ -0,0 +1,9 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  tasks:
+  - name: Example assertion
```

Comment:
I assume actually running wireguard and creating network devices won't work in docker without additional modifications but we could check if the config file exists

Reply:
That's what we want to test in the multiserver scenario, with that scenario we can get to test most of the OS supported by the role.